reconcile/terraform_vpc_peerings.py

```diff
@@ -454,8 +452,9 @@ def run(dry_run, print_only=False,
         sys.exit(1)
 
     if dry_run:
-        return
-
-    err = tf.apply()
-    if err:
+        sys.exit(0)
```

Comment:
according to my comments above, i think we should exit with any(errors) and not with `0`.

Reply:
in general, as we are only focusing on dry-run, i don't think we should change this block of code.

Reply:
Just for my understanding, do we want to call `tf.plan` under errors?

* If `dry_run` ?
* If `not dry_run` ?

The current version doesn't plan if there are errors, `dry_run` or `not dry_run`. I'm getting confused here.